app/poros/directions.rb

```diff
@@ -0,0 +1,9 @@
+class Directions
+  attr_reader :travel_time,
+              :instructions
+
+  def initialize(data)
+    @travel_time = data.first[:formattedTime]
```

Comment:
Something to double check with your logic here -- are you taking the `formattedTime` from each maneuver or for the whole trip? In your facade, you're mapping the Mapquest directions JSON and only taking the `leg[:maneuvers]` collection, so by taking the travel time just from this first hash, I belive you're just grabbing the amount of time it should take to complete only the first maneuver. 

Another option:
-pass the whole directions JSON into a PORO
- In the PORO, grab the travel time using `json_data[:route][:formatted_time]` (which in this case is the same as the formatted time in legs.first)
- In the PORO, grab the maneuvers using `json_data[:route][:legs].first[:manuevers].map { |man| man[:narrative] }.flatten`

There are a lot of ways to concatenate the direction narratives in different places but the important piece is grabbing the right travel time from the request